docs/online-boutique-asm-manifests/base/for-memorystore/kustomization.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Sample to illustrate how to customize the OnlineBoutique's cartservice to use Memorystore (redis) instead of in-cluster redis container, you need to replace:
+# REDIS_IP and REDIS_PORT in this current file before including this file in another kustomization.yaml file.
+# As an example:
+# sed -i "s/REDIS_IP/0.0.0.0/g;s/REDIS_PORT/6379/g" kustomization.yaml
+
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+patchesJson6902:
+- target:
+    kind: Deployment
+    name: cartservice
+  patch: |-
+    - op: replace
+      path: /spec/template/spec/containers/0/env/0
+      value:
+        name: REDIS_ADDR
+        value: REDIS_IP:REDIS_PORT
```

Comment:
I didn't think about that, that's a good point. Now I think more about this, Memorystore has 2 flavors: Memcache and Redis and I'm not sure `cartservice` will work with Memcache (?). Furthermore, it could be any Redis database, hosted elsewhere, like in a VM, etc.  So I think keeping this `REDIS_` prefix generic still makes more sense, isn't it?

Note: actually the value of `REDIS_IP` could be an host/fqdn, so wondering if we want to rename `REDIS_IP` by `REDIS_HOST` to be more generic? Memorystore (redis) only supports IP address anyway... so maybe we keep this like this?